<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monthlyincomepage</class>
 <widget class="QDialog" name="monthlyincomepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select month:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select year:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Credit monthly:</string>
   </property>
  </widget>
  <widget class="QLabel" name="monthlyCredit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Cash monthly:</string>
   </property>
  </widget>
  <widget class="QLabel" name="monthlyCash">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total monthly Incomes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="monthSelect">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="yearSelect">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2020</number>
   </property>
   <property name="maximum">
    <number>2021</number>
   </property>
  </widget>
  <widget class="QLabel" name="monthlyIncomes">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectedButton">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>80</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
